msteams-platform/concepts/build-and-test/share-to-teams.md

```diff
@@ -7,7 +7,7 @@ keywords: Share Teams Share-to-Teams
 ---
 # Create Share-to-Teams button
 
-Third-party websites can use the launcher script to embed Share-to-Teams buttons on their webpages. When you select, it launches the Share-to-Teams experience in a pop-up window. This allows you to share a link directly to any person or Microsoft Teams channel without switching the context. This document guides you on how to create and embed a Share-to-Teams button for your website, craft your website preview, and extend Share-to-Teams for Education.
+Third-party websites can use the launcher script to embed Share-to-Teams buttons on their webpages. When you select, it launches the Share-to-Teams experience in a pop-up window. This allows you to share a link directly to any person or Microsoft Teams channel without switching the context. You can also add link unfurling for the shared link. For more information, see [link unfurling](~/messaging-extensions/how-to/link-unfurling.md). This document guides you on how to create and embed a Share-to-Teams button for your website, craft your website preview, and extend Share-to-Teams for Education.
```

Comment:
When you select, it launches - Select what?

Reply:
"This document guides you" - is not the standard. Please change it as per the standard